package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "4.5.2",
+  "version": "5.0.0",
```

Comment:
I do not think we are prepared for the extra work implied to support a breaking change.

Reply:
Will update according to https://github.com/hubiinetwork/nahmii-sdk/pull/161#discussion_r528647388_.